.github/workflows/ci.yml

```diff
@@ -42,7 +42,9 @@ jobs:
         run: sudo apt-get update && sudo apt-get install --no-install-recommends -y build-essential git libpq-dev libyaml-dev pkg-config
 
       - name: Checkout code
-        uses: actions/checkout@v4
+      - uses: actions/checkout@v5
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
does this run on main, and if so, what'll its `github.event.pull_request.head.sha` be?

Reply:
Good catch! I realize now also that the default behavior (merging main with the PR) accomplishes (without explaining itself, which is annoying) the task we were doing manually in vita-min of making sure that your PR doesn't break any tests which have been added to main since you branched, so we're restoring it.